draft-ietf-ppm-dap.md

```diff
@@ -152,6 +152,16 @@ input is ever seen in the clear by any aggregator.
 
 12:
 
+- Bump draft-irtf-cfrg-vdaf-08 to 12 {{!VDAF}}. (\*)
+
+- Add support for "asynchronous" aggregation, based on the Leader polling the
+  Helper for completion. (\*)
+
+- Update collection semantics to match the new aggregation semantics introduced
+  in support of asynchronous aggregation. (\*)
+
+- Clarify the requirements around report replay protection.
+
 - Remove the `max_batch_size` parameter of the fixed-size query type.
 
 - Rename the "fixed-size" query type to "leader-selected", to align the name
```

Comment:
nit: I would swap the order of these: mention renaming first, and then say how the renamed thing has changed.